### PR TITLE
feat(components/atom/loading): Add atom loading component

### PR DESCRIPTION
**Pull request by oegea**
### Types of changes

- [X] ✨ New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Review by davidmartin84 — APPROVED** [review]

**Comment by andresin87** [comment]
wait

**Review by jordevo — APPROVED** [review]

File: components/atom/loading/src/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types'
+
+const AtomLoading = ({role = 'status', ...props}) => {
+  return <div role="status" {...props} className="sui-AtomLoading" />
```
Comment (oriolpuig) [inline components/atom/loading/src/index.js]: We have defined a default value for `role` prop but we are not using it inside, right?

```suggestion
  return <div role={role} {...props} className="sui-AtomLoading" />
```

File: components/atom/loading/test/index.test.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    // When
+    const {container} = setup(props)
+
+    // Then
+    expect(container.innerHTML).to.be.a('string')
+    expect(container.innerHTML).to.not.have.lengthOf(0)
+  })
+
+  it.skip('should NOT extend classNames', () => {
```
Comment (oriolpuig) [inline components/atom/loading/test/index.test.js]: Can we remove or unskip this test?

**Review by oriolpuig — APPROVED** [review]
👌🏻 I don't know if make sense to use this `AtomLoading` inside `AtomButton` to reuse the code and components instead of have the logic duplicated 🤔

**Comment by oegea** [comment]
> 👌🏻 I don't know if make sense to use this `AtomLoading` inside `AtomButton` to reuse the code and components instead of have the logic duplicated 🤔

Yes! We haven't included it yet. First we want to create the component and release it, and then create a different pr to requiring it from the atom button. We are doing this in two steps just to avoid CI problems when submitting changes.

**Review by andresin87 — COMMENTED** [review]
✅

**Pull request merged**